css/style.css

```diff
@@ -0,0 +1,2886 @@
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 420px 0;
+  }
+
+  .top__wrapper-item {
+    margin: 30px 0;
+    display: flex;
+    justify-content: center;
+    margin-top: 45px;
+    margin-bottom: 60px;
+    margin-left: 15px;
+  }
+
+  .charset {
+    margin: 0 30px;
+    justify-content: center;
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 92px;
+  }
+
+  .charset__road {
+    display: none;
+    width: 37.83px;
+    height: 60.72px;
+    margin-left: 20px;
+  }
+
+  .charset__first-big {
+    font-weight: 600;
+    font-size: 50px;
+    padding-left: 15px;
+    padding-bottom: 5px;
+  }
+
+  .charset__first-small {
+    margin-right: 62px;
+    padding-right: 0;
+    padding-left: 7px;
+    padding-bottom: 10px;
+  }
+
+  .button {
+    margin: 0;
+    justify-content: center;
+  }
+
+  .button__outline {
+    color: #e51937;
+    padding-bottom: 10px;
+    padding-top: 16px;
+    padding-right: 40px;
+    padding-left: 45px;
+    background: url(../img/svg/steering-ipad.svg) no-repeat;
+    background-position: 0 6px;
+  }
+
+  .button__outline-img {
+    display: none;
+    padding-left: 28px;
+    margin-right: 15px;
+    margin-left: 0;
+  }
+
+  .button__text {
+    width: 110px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .button__red {
+    background-color: #e51937;
+    color: #fff;
+    margin: 0 30px;
+    justify-content: center;
+    margin-left: 0;
+    padding-left: 45px;
+    padding-right: 45px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    margin-right: 30px;
+    margin-bottom: 0;
+  }
+
+  .middle__inner {
+    max-width: 800px;
+  }
+
+  .middle__fon {
+    max-width: 800px;
+  }
+
+  .middle__info-row {
+    max-width: 800px;
+  }
+
+  .middle__row {
+    margin-left: 0;
+  }
+
+}
+
+@media (max-width: 768px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .header {
+    padding-right: 0;
+    padding-left: 0;
+    margin-right: 0;
+  }
+
+  .header__logo {
+    padding-left: 15px;
+  }
+
+  .header__right {
+    margin-right: 33px;
+  }
+
+  .header__burger-img {
+    width: 35px;
+    height: 22px;
+  }
+
+  .top {
+    padding: 90px 0 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 177px;
+    background-repeat: no-repeat;
+    background-size: 822px 271px;
+  }
+
+  .top__title {
+    margin-left: 0;
+    margin-top: 55px;
+  }
+
+  .top__subtitle {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .top__img {
+    width: 700px;
+    height: auto;
+    margin-right: 0;
+    margin-top: 89px;
+    margin-left: 0;
+  }
+
+  .top__wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-left: 10px;
+    margin-top: 44px;
+  }
+
+  .top__wrapper-charset {
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 413px 1px;
+    margin-top: 0;
+  }
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)